managers/templates/managers/partials/display_project_expected_hours.html

```diff
@@ -0,0 +1,31 @@
+{% load data_display_filters %}
+
+{% regroup object.get_expected_hours_ordered by user as expected_hours_for_user %}
+    <div class="table">
+        {% for user in expected_hours_for_user %}
+            <table class="table">
+                <thead>
+                    <h4 title="{{ user.grouper.get_full_name  }}">
```

Comment:
What does `grouper` do?

Reply:
It's django attribute that works with regroup:
https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#regroup